client/web/src/settings/AppSettingsSubform.js

```diff
@@ -45,13 +51,24 @@ export default class AppSettingsSubform extends React.Component {
                       type="text"
                       disabled={this.props.isLocked}
                     />
-                    <SaasBoostInput
-                      key="sslCertificate"
-                      label="SSL Certificate ARN"
+                    <SaasBoostSelect
+                      type="select"
                       name="sslCertificate"
-                      type="text"
-                      disabled={this.props.isLocked}
-                    />
+                      id="sslCertificate"
+                      label="ACM SSL Certificate"
```

Comment:
Could be more user friendly to have this label be "SSL Certificate" and have it be a hyperlink to the Certificate Manager console https://us-west-2.console.aws.amazon.com/acm/home?region=us-west-2#/certificates/list (obviously will have to dynamically set the region in that URL)